### PR TITLE
I900 forgot password flow

### LINKED ISSUE #900: Redirect Forgot Password to Restore Vault Screen
Currently our 'Forgot Password' link takes us to the landing page, but if someone has legit forgot their password, it's unlikely that they'd make new vaults if they had any amount of ether in their wallets.

### PR DESCRIPTION
**Pull request by 2-am-zzz**
fixes #900 by redirecting forgotten password prompts to seed word screen.

### CODE REVIEW

File: app/scripts/keyring-controller.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ module.exports = class KeyringController extends EventEmitter {
   // or accept a state-resolving promise to consume their results.
   //
   // Not all methods end with this, that might be a nice refactor.
-  fullUpdate() {
+  fullUpdate () {
```
Comment (flyswatter) [inline app/scripts/keyring-controller.js]: If you want to make this a rule, you should just make it a hard fail in the `.eslintrc` file, I feel bad with you always patching up the warnings alone.

**Review by danfinlay — COMMENTED** [review]
This is probably fine, but I want to see it in action before I sign off.

File: ui/app/unlock.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ UnlockScreen.prototype.render = function () {
 
       h('.flex-row.flex-center.flex-grow', [
         h('p.pointer', {
-          onClick: () => this.props.dispatch(actions.showRestoreVault()),
+          onClick: () => this.props.dispatch(actions.forgotPassword()),
```
Comment (flyswatter) [inline ui/app/unlock.js]: An easier way to have done this whole commit would have been to add a line of `this.props.dispatch(actions.transitionForwards())` before `showRestoreVault`.

That said, you did this fast, thank you!

**Review by danfinlay — APPROVED** [review]
Looks good!

**Pull request merged**